skeletons/constr_SET_OF_jer.c

```diff
@@ -79,8 +79,7 @@ SET_OF_decode_jer(const asn_codec_ctx_t *opt_codec_ctx,
             asn_dec_rval_t tmprval = {RC_OK, 0};
 
             /* Invoke the inner type decoder, m.b. multiple times */
-            ASN_DEBUG("JER/SET OF element [%s]", elm_tag(*element->name) ? 
-                      element->name : element->type->xml_tag);
```

Comment:
I'd prefer to see the element name here - either its value, or printed `"NULL"` if empty.